core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -845,6 +848,20 @@ public static ExceptionVerbosity annotatedServiceExceptionVerbosity() {
         return ANNOTATED_SERVICE_EXCEPTION_VERBOSITY;
     }
 
+    /**
+     * Enables {@link DefaultAddressResolverGroup} that resolves domain name using JDK's built-in domain name
+     * lookup mechanism.
+     * Note that JDK's built-in resolver performs a blocking name lookup from the caller thread, and thus
+     * this flag should be enabled only when the default asynchronous resolver does not work as expected.
```

Comment:
Maybe can add something like `, for example by always throwing a {@code FooException}.`